haskell-indexer-backend-ghc/src/Language/Haskell/Indexer/Backend/Compat.hs

```diff
@@ -299,15 +311,15 @@ needsTemplateHaskellOrQQ = needsTemplateHaskell
 mgModSummaries = id
 #endif
 
+{-# COMPLETE ValBindsCompat #-}
```

Comment:
This COMPLETE doesn't seem right to me; what about the case of ValBinds for HsValBindsLR?  Or am I reading it wrong?
http://hackage.haskell.org/package/ghc-8.6.5/docs/HsBinds.html#t:HsValBindsLR

Reply:
Yeah, I was cheating here :)

It makes the code complicated to remove warnings across multiple ghc versions because those "X"-prefixed data constructors were added in ghc 8.6.x.

I can add an additional pattern case in the client code for `XValBindsLR` only for 8.6.x, but all I can do there is erroring out. I'm not sure how much better that is compared to this because both will error and crash anyway.

Let me think about this more.

Reply:
Fixed by following the same pattern as the <806 case.